.eslintrc.js

```diff
@@ -35,5 +35,6 @@ module.exports = {
     '@typescript-eslint/camelcase': 0,
     '@typescript-eslint/explicit-function-return-type': 0,
     '@typescript-eslint/no-var-requires': 0,
+    '@typescript-eslint/explicit-member-accessibility': 0,
```

Comment:
This negates the first rule above (where it's set to 1) so if we'd like to change let's discuss